scripts/bungie.coffee

```diff
@@ -195,8 +233,8 @@ getCharacterId = (bot, membershipType, playerId, robot) ->
 
   deferred.promise
 
-# Gets itemInstanceId from Inventory Summary based on weaponSlot
-getItemIdFromSummary = (bot, membershipType, playerId, characterId, weaponSlot) ->
+# Gets itemInstanceId from Inventory Summary based on bucket
+getItemIdFromSummary = (bot, membershipType, playerId, characterId, bucket, robot) ->
```

Comment:
Because the code was attempting to send a message to the room if anything was wrong, but there was no `robot` passed to that method:
```
    item = items.filter(matchesBucketHash)
    if item.length is 0
      robot.send {room: bot.message.user.name, "unfurl_media": false}, "Something went wrong, couldn't find the requested item for this character. #{helpText}"
```